addons/udes_stock/models/stock_picking_batch.py

```diff
@@ -618,6 +618,17 @@ def is_valid_location_dest_id(self, location_ref):
         return all([pick.is_valid_location_dest_id(location=location)
                     for pick in all_done_pickings])
 
+    def _check_unpickable_item(self):
+        """ Checks if the picking type of the batch is allowed to handle
+            unpickable items.
+            If it is not, it raises an error asking to contact team leader.
+        """
+        if not all(self.picking_type_ids.mapped('u_handle_unpickable_items')):
```

Comment:
Ofc, you gotta invert the condition, but it's always good to get rid of the `not` in a clause.

Reply:
I was going to change it but having to invert the condition makes it more confusing. I have left the original statement since it matches to what we are semantically doing:
If not all picking types can handle unpickable items: bla bla bla.